### PR TITLE
Script for preparing calibration and test sets 

### PR DESCRIPTION
**Pull request by fatihcakirs**
Related issue is https://github.com/mlcommons/mobile_app_open/issues/108. 

Work-in-progress. ADE20K and COCO datasets are done. 

The script prepares and outputs the test sets to `mlperf_datasets` folder, which can directly be copied to a device for accuracy evaluation (excludes any ground truth files). I've used the processing scripts provided by MLPerf.

Calibration sets are outputted to `calibration_set` folder. Calibration images are not processed. Per MLPerf rules, the calibration images are not subject to the same processing requirements as with the test images, therefore it is best to leave them as is for any processing by submitters imo.

### CODE REVIEW

File: datasets/Makefile [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+${BUILDDIR}/imagenet.stamp:
+	@echo "ImageNet not implemented"
+	@touch $@
+
+${BUILDDIR}/squad.stamp:
+	@echo "Squad not implemented"
+	@touch $@
+
+
+clean:
```
Comment (mcharleb) [inline datasets/Makefile:163]: These are just placeholders?

File: datasets/util/ade20k/ade20k_calibration_files.txt [hunk 2]
```diff
@@ -0,0 +1,500 @@
+ADE_train_00002043.jpg
```
Comment (mcharleb) [inline datasets/util/ade20k/ade20k_calibration_files.txt:1]: What is the license of this file, and where did it come from? That should be documented somewhere.

File: datasets/util/coco/coco_calibration_files.txt [hunk 3]
```diff
@@ -0,0 +1,500 @@
+000000391895.jpg
```
Comment (mcharleb) [inline datasets/util/coco/coco_calibration_files.txt:1]: What is the license of this file, and where did it come from? That should be documented somewhere.

**Review by mcharleb — CHANGES_REQUESTED** [review]

**Comment by mcharleb** [comment]
@fatihcakirs  I cannot add changes to your PR because you pushed it from your personal github repo vs a branch on mobile_app_open.

**Comment by mcharleb** [comment]
@fatihcakirs When I run "make" after cloning your repo I get:

```
...
Successfully tagged mlperf_mobile:1.1
Generating ADE20K calibration set
make: *** No rule to make target '/local/mnt2/workspace2/mcharleb/mlperf/fatih/datasets/output/calibration_set/ade20k/images/ADE_train_00002043.jpg', needed by '/local/mnt2/workspace2/mcharleb/mlperf/fatih/datasets/ade20k_calibration.stamp'.  Stop.
```

**Comment by fatihcakirs** [comment]
Thanks @mcharleb for the review. I've re-checked and it should work. In the mean time, I've created PR https://github.com/mlcommons/mobile_app_open/pull/127 but from a mobile_app_open branch . We can continue the discussion over there if you like. Closing this one.

**Pull request closed without merging**